Package.swift

```diff
@@ -26,19 +26,17 @@ var targets = [
 	),
 ]
 
-var platforms: [SupportedPlatform] = [] // leave blank for more compatibility with swiftpackageindex?
-
-#if canImport(AppleProductTypes) // swift package dump-package fails because of this
-import AppleProductTypes
-
-platforms += [ // minimums for Date.now
+var platforms: [SupportedPlatform] = [ // minimums for Date.now
 	.iOS("15.2"),
 	.macOS("12.0"),
 	.tvOS("15.0"),
 	.watchOS("8.0"),
 	.visionOS("1.0"),
 ]
```

Comment:
NB: the `platforms` key does _not_ indicate which platforms a package supports, despite its name. What it does is set minimum required versions for certain platforms. The absence of a platform here simply means it has _no_ restrictions, not that it's not compatible.